iothub/service/src/Jobs/ScheduledDeviceMethod.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains fields used when creating a job to run a device method on one or multiple devices.
+    /// </summary>
+    public class ScheduledDeviceMethod
+    {
+        /// <summary>
+        /// Query condition to evaluate which devices to run the job on.
+        /// </summary>
+        [JsonProperty(PropertyName = "QueryCondition", Required = Required.Always)]
```

Comment:
I'm assuming the JSON property name should be camelCased, and the service JSON is always (AFAIK) cased as such.